src/sentry/static/sentry/app/actionCreators/members.jsx

```diff
@@ -14,6 +14,16 @@ export function fetchOrgMembers(api, orgId) {
   });
 }
 
+export function fetchProjectMembers(api, orgId, projectId) {
+  const url = `/projects/${orgId}/${projectId}/members/`;
+  return api.requestPromise(url, {method: 'GET'}).then(members => {
+    members = members.filter(m => m.user).map(m => m.user);
+    MemberListStore.loadInitialData(members);
```

Comment:
Is there a reflux action that should be fired instead of calling the store directly?

Reply:
There aren't any actions for the MemberListStore. It is an odd bit of global state. Personally I'd like to replace it. The assumptions it makes are not working well with the move to organization based views.